packages/constellation/package.json

```diff
@@ -105,7 +105,7 @@
     "prettier-config-constellation": "workspace:*",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-test-renderer": "18.2.0",
+    "react-test-renderer": "17.0.2",
```

Comment:
@noahub This change in particular might be an issue. When adding back in our (non component) utils, hooks, services i ran into an issue when running the tests. It turned out to be because react-test-renderer was a different version than react so I downgraded it. Since it was a major version downgrade will this effect anything?

Reply:
for context https://stackoverflow.com/questions/72914208/typeerror-0-reacttestrenderer-act-is-not-a-function

Reply:
@blockdylanb I don't recall needing a specific version of react test renderer (I think the only reason it's `18.2.0` is because that's the most recent version). So I think as long as we're all green it should be good to go 👍